docs/getting-started.rst

```diff
@@ -1,7 +1,13 @@
 Getting Started With ODK
 =========================
 
-This document walks you through the recommended workflow to get started with ODK.
+ODK lets you build powerful offline forms to collect the data you need wherever it is.
+
+The most common use-case is a data collector fills out a form using the ODK Collect app on a phone or tablet. When that device finds an Internet connection it automatically sends the form data.
```

Comment:
"use-case" doesn't feel right to me. Usage scenario? Usage pattern? But also not terrible if you just want to leave it.